### PR TITLE
Ensure function name isn't mangled, grid size stays >0

### PR DESCRIPTION
**Pull request by TristanDebrunner**
Fixes to #42.

### CODE REVIEW

**Comment by TristanDebrunner** [comment]
Not sure why these didn't show up when I was testing #42 originally. I suspect cargo wasn't actually building with the cuda feature, because the build fails without these changes, but I have no idea why it was building without cuda.

File: src/cuda-poh-verify/poh_verify.cu [hunk 1]
```diff
@@ -124,6 +125,7 @@ int poh_verify_many(const uint8_t* hashes,
     CUDA_CHK(cudaMemcpyAsync(cur_ctx->num_hashes_arr, num_hashes_arr, num_hashes_size, cudaMemcpyHostToDevice, stream));
 
     int num_blocks = ROUND_UP_DIV(num_elems, NUM_THREADS_PER_BLOCK);
+    num_blocks += num_blocks == 0 ? 1 : 0;
```
Comment (sakridge) [inline src/cuda-poh-verify/poh_verify.cu]: Can you just bail if num_elems == 0 earlier?

Comment (TristanDebrunner) [reply src/cuda-poh-verify/poh_verify.cu]: Good question, I was seeing the problem even with what should have been `num_blocks=1`, need to check why that was

Comment (TristanDebrunner) [reply src/cuda-poh-verify/poh_verify.cu]: nm, it's just a problem with num_elems==0

**Pull request merged**